src/waveform/renderers/allshader/waveformrendererstem.cpp

```diff
@@ -121,7 +135,7 @@ bool WaveformRendererStem::preprocessInner() {
 
     const int numVerticesPerLine = 6; // 2 triangles
 
-    const int reserved = numVerticesPerLine * (8 * pixelLength + 1);
+    const int reserved = numVerticesPerLine * (8 * stripLength + 1);
```

Comment:
```suggestion
    const int reserved = numVerticesPerLine * (mixxx::audio::ChannelCount::stem() * stripLength + 1);
```
Did I guess right, where 8 comes from?